src/components/Counter/__tests__/__snapshots__/counter-snapshot-tests.jest.tsx.snap

```diff
@@ -176,13 +193,15 @@ exports[`Counter renders correctly with the prefix 1`] = `
 <span
   aria-label=" 13"
   aria-labelledby=""
+  className=""
```

Comment:
this should probably either not appear at all or contain something

Reply:
Removed